### ISSUE TITLE
train loss and val loss

### ISSUE THREAD

**Issue by JasperKirk** [RangiLyu/nanodet#77]
Thanks for your work.
When I use my own data for training, the Epoch training loss keeps decreasing, but the Epoch val loss is increasing. Is there any solution?
![image](https://user-images.githubusercontent.com/32267033/102007507-a50c4200-3d64-11eb-9c48-52a3ff53c6c5.png)

**Comment by RangiLyu** [comment]
This is weird. Maybe there is an error in the data annotation loading. Try to check that the training and validation data is loaded correctly.
